include/mapbox/geojsonvt.hpp

```diff
@@ -39,7 +39,7 @@ struct Options {
 const Tile empty_tile{};
 
 inline uint64_t toID(uint8_t z, uint32_t x, uint32_t y) {
-    return (((1 << z) * y + x) * 32) + z;
+    return (((1ull << z) * y + x) * 32) + z;
```

Comment:
`ull` stands for `unsigned long long` (usually 64-bit) and `u` is simply `unsigned`(usually 32-bit). These are used to tell the compiler we want those numbers to be contained in types that can hold them. In this specific case we use 64-bits because we return a `uint64_t` type.